packages/imperative/src/config/cmd/init/init.handler.ts

```diff
@@ -164,14 +164,14 @@ export default class InitHandler implements ICommandHandler {
     }
 
     /**
-     * Prompts for the value of a property on the CLI. Returns null if `--ci`
+     * Prompts for the value of a property on the CLI. Returns null if `--prompt false`
      * argument is passed, or prompt times out, or a blank value is entered.
      * @param propName The name of the property
      * @param property The profile property definition
      */
     private async promptForProp(propName: string, property: IProfileProperty): Promise<any> {
         // skip prompting in CI environment
-        if (this.arguments.ci) {
+        if (this.arguments.prompt === false) {
```

Comment:
can we revisit this on standup?  i'm wondering about confusion with using `prompt=false` and i kind of liked the simpler flag `--ci`.  OR, how about something like `--no-prompt` ?

Reply:
`--no-prompt` is also supported. For a Boolean arg `--x`, Yargs will automatically convert `--no-x` to `--x false`

Reply:
![image](https://user-images.githubusercontent.com/14940561/103907431-4326ca00-50cf-11eb-9db7-0a46c939de7d.png)

Reply:
![giphy](https://user-images.githubusercontent.com/14940561/103907513-5afe4e00-50cf-11eb-8627-6465112f3e27.gif)

Reply:
i never knew this!